src/metabase/driver/postgres.clj

```diff
@@ -301,7 +301,7 @@
                        (qp.store/field id-or-name))
         parent-method (get-method sql.qp/->honeysql [:sql :field])
         identifier    (parent-method driver clause)
-        nfc-path      (:nfc_path stored-field)]
+        _nfc-path     (:nfc_path stored-field)]
```

Comment:
these unused bindings made me nervous @howonlee . Just want to make sure nothing got lost in a merge/rebase or whatever.

Reply:
I swear to god I've rewritten that bit 25 times. nothing got missed except me deleting the binding, you can leave it in if you'd like